esp/tools/soapplus/main.cpp

```diff
@@ -513,7 +513,12 @@ int main(int argc, char** argv)
         else if (stricmp(argv[i], "-v")==0)
         {
             i++;
-            doValidation = true;
+            doValidation = 1;
+        }
+        else if (stricmp(argv[i], "-v2")==0)
```

Comment:
So -v really means "schema validate the result", and this new option means "validate xml formatting".

If this was a newer tool I would use a verbose name... but to remain consistent with the soapplus short names, how about "-vx" (meaning "validate xml")?  Not a big fan of numbers.

Also can we add a description to the usage?